unidist/cli/base/utils.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (C) 2021 Modin authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""Utilities for backend runners."""
+
+import os
+import ipaddress
+import socket
+
+from unidist.config import CpuCount, Backend
+
+
+class Defaults:
+    """Default values for supported CLI parameters."""
+
+    BACKEND = Backend.get()
+    NUM_CPUS = CpuCount.get()
+    HOSTS = "localhost"
+    REDIS_PASSWORD = ""
```

Comment:
Should Ray default value be used?

Reply:
No, because in this case we'll include a hard dependency on Ray API for all our backends.